pkg/base1/test-channel.ts

```diff
@@ -0,0 +1,538 @@
+import QUnit from "qunit-tests";
+
+import { transport_globals } from 'cockpit/_internal/transport';
+import { Channel, ChannelControlMessage, ChannelOptions, ChannelPayload } from 'cockpit/channel';
+
+// simple function to 'use' values (to avoid 'unused' warnings)
+function use(...args: unknown[]) {
+    console.log(args.length);
```

Comment:
That makes running the unit test unnecessarily noisy. Is there no better way to do this, and just label the function argument `_args`? You set up some eslint rules in the beginning of the series to ignore variables prefixed with `_`, no?

Reply:
Ah right, I did all of this before the eslint changes.

fwiw, this never runs.  It's all part of the static type checking which doesn't actually execute.

Reply:
Ah, thanks. so if we can use `_var` I think that'd look a bit cleaner, otherwise ok.

Reply:
So I looked into this.  For function arguments we can indeed use `_`, so this lets us avoid the `console.log()` call.  For local variables, TypeScript itself complains about the useless variable.  It does it as a warning, so you don't get it at `test/static-code` type, but it clutters up my editor, so better not.  I could probably configure it, but eh... There are also weird effects of not assigning to a variable like "don't use `new` for side-effects" and "useless constructor".  All in all, `use()` is just more convenient for a lot of cases, so I'm gonna keep it, but avoid it for the cases where I can...

Reply:
Right, thanks for checking!